packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.5.0]
+### Added
+- Add `KeyringController` messenger actions ([#1691](https://github.com/MetaMask/core/pull/1691))
+  - `KeyringController:getAccounts`
+  - `KeyringController:getKeyringsByType`
+  - `KeyringController:getKeyringForAccount`
+
```

Comment:
We should mention #1669 here as well. It's not breaking, but it does change the package manifest

```suggestion

### Changed
- Bump `@metamask/eth-sig-util` from 6.0.0 to 7.0.0 ([#1669](https://github.com/MetaMask/core/pull/1669))

```